pydantic/schema.py

```diff
@@ -277,7 +277,7 @@ def get_field_schema_validations(field: ModelField) -> Dict[str, Any]:
         f_schema['const'] = field.default
     if field.field_info.extra:
         f_schema.update(field.field_info.extra)
-    modify_schema = getattr(field.outer_type_, '__modify_schema__', None)
```

Comment:
can we delete this file? Would make me so happy.

Reply:
Lol yes I'll do that